fish-pepper/jolokia/fp-files/jolokia-opts

```diff
@@ -13,65 +10,83 @@ is_in_jolokia_opts() {
   fi
 }
 
+get_jolokia_properties() {
+
+  echo "host=${AB_JOLOKIA_HOST:-*}"
+  echo "port=${AB_JOLOKIA_PORT:-8778}"
+  echo "discoveryEnabled=false"
```

Comment:
Why should this be always the case ? I see the point for OpenShift but I'm not sure that we should this rule out for everybody. 'would be fine for me to introduce a 'AB_JOLOKIA_DISCOVERY_ENABLED' and only switch in on when set.